app/routes/session.py

```diff
@@ -163,12 +165,11 @@ def get_sign_out():
     """
     survey_config = get_survey_config()
     log_out_url = (
-        survey_config.account_service_todo_url if "todo" in request.args else None
+        url_for("session.get_signed_out")
+        if "save" in request.args
```

Comment:
Maybe `external` would be clearer?

Reply:
I originally put external in after we had that chat but save made more sense to me because we are saving their answers. I wasn't sure what you meant by external

Reply:
As in the redirect is `external` or not. Save to me was confusing as we are not saving at this point, we are only redirecting. Save only happens on POST requests. I'm not fussed on `external` but something better than `save` as it is confusing as it makes you think it is saving when it is not.

Reply:
Ok Ill update it to be `external-redirect` to make it more explicit